marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx

```diff
@@ -305,3 +305,34 @@ describe('json scenario for table on browse documents page', () => {
   });
 
 });
+
+
+describe('Verify numeric facet can be applied', () => {
+    //login with valid account and go to /browse page
+    beforeEach(() => {
+        cy.visit('/');
+        cy.contains(Application.title);
+        cy.loginAsTestUserWithRoles("pii-reader").withUI();
```

Comment:
any reason to use withUI here than withRequest?
Also can we add an afterEach to reset test user so begin state of a new test is same always.

Reply:
@bsrikan I have changed the login to withRequest, Scenarios.json has three test suites in total and each test suite has a seperate beforeEach which logs in with required roles, not sure how afterEach is helpful ?

Reply:
You are right, maybe not needed if there is only one test. Its just a safer approach.